views/js/qtiCreator/editor/response/choiceSelector.js

```diff
@@ -16,39 +16,110 @@
  * Copyright (c) 2016 (original work) Open Assessment Technologies SA;
  *
  */
-define(['jquery',
+define([
+    'jquery',
     'lodash',
     'i18n',
     'ui/component',
-    'tpl!taoQtiItem/qtiCreator/tpl/toolbars/choiceSelector'
+    'tpl!taoQtiItem/qtiCreator/tpl/toolbars/choiceSelector',
+    'select2'
 ], function($, _, __, component, choiceSelectorTpl){
-    
-    //exposed methods
+
+    'use strict';
+
+    /**
+     * Exposed methods
+     * @type {{getChoices: choiceSelector.getChoices, getSelectedChoices: choiceSelector.getSelectedChoices}}
+     */
     var choiceSelector = {
-        getChoices : function(){},
-        setChoices : function(){},
-        resetChoices : function(){}
+        getChoices : function(){
+            return this.config.interaction.choices || {};
```

Comment:
I recommend using an internal private variable to store the select choices. Adding it to configu make this public + it is no longer a config.
I ould have the same recommendation for getSelectedChoices() and setSelectedChoices();